### PR TITLE
docs: Install parameters

### PR DESCRIPTION
**Pull request by sobi3ch**
Mandatory awsRegion and optional awsAccessKeyID and awsSecretAccessKey

### CODE REVIEW

**Comment by sobi3ch** [comment]
I spend over an hour to discover I've used [helm install example](https://github.com/kubernetes/autoscaler/tree/master/charts/cluster-autoscaler#aws---using-auto-discovery-of-tagged-instance-groups) without AWS region and realize this reading [this](https://github.com/kubernetes/autoscaler/issues/2936#issuecomment-617289741) comment. I believe this will be useful update to the docs.

File: charts/cluster-autoscaler/README.md [hunk 1]
```diff
@@ -81,10 +81,16 @@ Auto-discovery finds ASGs tags as below and automatically manages them based on
...
 - Set `awsRegion=<YOUR AWS REGION>`
-- Set `awsAccessKeyID=<YOUR AWS KEY ID>` and `awsSecretAccessKey=<YOUR AWS SECRET KEY>` if you want to [use AWS credentials directly instead of an instance role](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md#using-aws-credentials)
+- Set (option) `awsAccessKeyID=<YOUR AWS KEY ID>` and `awsSecretAccessKey=<YOUR AWS SECRET KEY>` if you want to [use AWS credentials directly instead of an instance role](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md#using-aws-credentials)
 
 ```console
-$ helm install my-release autoscaler/cluster-autoscaler --set autoDiscovery.clusterName=<CLUSTER NAME>
+$ helm install my-release autoscaler/cluster-autoscaler --set autoDiscovery.clusterName=<CLUSTER NAME> --set awsRegion=<YOUR AWS REGION>
+```
+
+Alternatively with your own AWS credencials
```
Comment (bpineau) [inline charts/cluster-autoscaler/README.md:90]: Thanks, the changes looks useful to me (I'm not a maintainer though), just noticed a small nit:
```suggestion
Alternatively with your own AWS credentials
```

**Comment by sobi3ch** [comment]
Thanks @bpineau

**Comment by gjtempleton** [comment]
Thanks for this, you're right, this is a useful addition to the docs.

/lgtm
/approve

**Pull request merged**